core/sequencer/sequencer_test.go

```diff
@@ -0,0 +1,97 @@
+// Copyright 2018 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package sequencer
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/google/keytransparency/core/mutator"
+	"github.com/google/keytransparency/core/mutator/entry"
+	"github.com/google/tink/go/signature"
+	"github.com/google/tink/go/tink"
+
+	tpb "github.com/google/trillian"
+)
+
+func queueMsg(id int64, signer *tink.KeysetHandle) *mutator.QueueMessage {
+	index := []byte{byte(id)}
+	userID := string(id)
+	m := entry.NewMutation(index, "domain", "app", userID)
+	signers := []*tink.KeysetHandle{signer}
+	pubkey, err := signer.GetPublicKeysetHandle()
+	if err != nil {
+		panic(fmt.Errorf("GetPublicKeysetHandle(): %v", err))
```

Comment:
Alternatively, you could pass in `t`, make this a `t.Helper()` and do `t.Fatal` here -- up to you.

Reply:
done